nomad/leader.go

```diff
@@ -425,6 +430,7 @@ func (s *Server) establishLeadership(stopCh chan struct{}) error {
 			go s.replicateACLAuthMethods(stopCh)
 			go s.replicateACLBindingRules(stopCh)
 			go s.replicateNamespaces(stopCh)
+			go s.replicateNodePools(stopCh)
```

Comment:
Technically no, because the replication token will just be the zero value empty string. But if you check out the federation tutorial and ACL tutorial it really looks like we consider federation and replication as two different problems. You can have federation without replication and that just forwards requests around. But the purpose of replication seems entirely focused on ACLs. Namespaces are arguably ACL objects, and in the ENT case so are Node Pools. So I think we should probably keep the behavior we have here with namespaces.